flink-core/src/main/java/org/apache/flink/configuration/ChangelogOptions.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.configuration;
+
+import org.apache.flink.annotation.docs.Documentation;
+import org.apache.flink.configuration.description.Description;
+
+import java.time.Duration;
+
+/** A collection of all configuration options that relate to changelog. */
+public class ChangelogOptions {
+
+    @Documentation.Section(Documentation.Sections.STATE_BACKEND_CHANGELOG)
+    public static final ConfigOption<Duration> PERIODIC_MATERIALIZATION_INTERVAL =
+            ConfigOptions.key("state.backend.periodic-materialize.interval")
```

Comment:
Add changelog prefix to key? (ditto other keys)
Maybe `state.backend.changelog.periodic-materialization.interval`?